ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomSheetDemoController.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+
+class BottomSheetDemoController: UIViewController {
+
+    override func loadView() {
+        view = UIView()
+
+        let optionTableView = UITableView(frame: .zero, style: .plain)
+        optionTableView.translatesAutoresizingMaskIntoConstraints = false
+        optionTableView.register(BooleanCell.self, forCellReuseIdentifier: BooleanCell.identifier)
+        optionTableView.register(ActionsCell.self, forCellReuseIdentifier: ActionsCell.identifier)
+        optionTableView.dataSource = self
+        optionTableView.delegate = self
+        optionTableView.separatorStyle = .none
+        view.addSubview(optionTableView)
+
+        let bottomSheetViewController = BottomSheetController(contentView: personaListView)
+        bottomSheetViewController.hostedScrollView = personaListView
+        bottomSheetViewController.expandedHeightFraction = 1.0
```

Comment:
technically we don't need this right? because by defulat it is 1

Reply:
yup